Makefile

```diff
@@ -1,26 +1,63 @@
 SHELL=/bin/bash
 
+#: build all the projects and subprojects
 build:
 	./gradlew build
 
+#: run the Java tests
 test:
 	./gradlew test
 
+project_version:=$(shell ./gradlew --quiet --console=plain project_version)
```

Comment:
```
Makefile:16: *** multiple target patterns.  Stop.
```

🤦🏻‍♂️  You know what `cimg/base` doesn't have? Java. It can't run this command to find out the project version.